source/r/check_presence.R

```diff
@@ -35,23 +34,20 @@ check_presence <- function(
   for (country in countries) {
     # Function to check occurrence data and handle errors
     check_occurrence <- function(name) {
-      data <- tryCatch(
-        {
-          rgbif::occ_data(
-            scientificName = name,
-            country = country, # Specify one country at a time
-            limit = 1
-          )
-        },
-        error = function(e) {
-          return(list(error = TRUE))
-        }
-      )
+      data <- tryCatch({
+        rgbif::occ_data(
+          scientificName = name,
+          country = country,  # Specify one country at a time
+          limit = 1
+        )
+      }, error = function(e) {
+        return(list(error = TRUE))
+      })
 
       if (inherits(data, "error") || is.null(data$data)) {
-        return(FALSE) # Occurrence data is not present or error occurred
+        return(FALSE)  # Occurrence data is not present or error occurred
```

Comment:
If you don't make a distinction between `error` and `no-data` then you might think a species doesn't occur in a country but actually there was just an error. (don't know how bad that is)